openedx_webhooks/info.py

```diff
@@ -25,6 +26,12 @@ def _read_repotools_yaml_file(filename):
     """Read a YAML file from the repo-tools-data repo."""
     return yaml.safe_load(_read_repotools_file(filename))
 
+def _read_repotools_csv_file(filename):
+    """
+    Read a CSV file from the repo-tools-data repo. Returns a list of rows.
```

Comment:
You could mention here that the first row is assumed to be headers which are skipped.
I know you don't want to fiddle with csv any more, but DictReader might be more convenient.